runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslator.java

```diff
@@ -318,6 +318,18 @@ public Job translate(List<DataflowPackage> packages) {
 
       WorkerPool workerPool = new WorkerPool();
 
+      // If streaming engine is enabled set the proper experiments so that it is enabled on the
+      // back end as well.
+      if (options.isEnableStreamingEngine()) {
```

Comment:
Add the enableStreamingEngine option if not present (when either experiment is present).
Only add an experiment if it is not present.

Reply:
Done